conda/recipes/dask-cudf/meta.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2018-2019, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
+{% set minor_version =  version.split('.')[0] + '.' + version.split('.')[1] %}
+{% set git_revision_count=environ.get('GIT_DESCRIBE_NUMBER', 0) %}
+{% set py_version=environ.get('CONDA_PY', 36) %}
+
+package:
+  name: dask-cudf
+  version: {{ version }}
+
+source:
+  path: ../../..
+
+build:
+  number: {{ git_revision_count }}
+  string: py{{ py_version }}_{{ git_revision_count }}
+
+requirements:
+  host:
+    - python
+    - cudf {{ version }}
```

Comment:
Should be {{ minor_version }}

Reply:
Given we're merging the repos won't it always be `{{ version }}` instead of `{{ minor_version }}` since if we hotfix we'll likely re-release dask-cudf as well?

Reply:
Disregard, this makes sense due to the merge in the case of hotfixes.